src/firewall.py

```diff
@@ -181,7 +181,7 @@ def delete_port(self, protocol: str, port: int | str) -> None:
     def apply(self, upnp: bool = True) -> bool:
         # FIXME: Ensure SSH is allowed
         ssh_port = _get_ssh_port()
-        if not self.config["tcp"][ssh_port]["open"]:
+        if ssh_port not in self.config["tcp"] or not self.config["tcp"][ssh_port].get("open", False):
```

Comment:
```suggestion
        if not self.config["tcp"].get(ssh_port, {}).get("open", False):
```
Is a more pythonic way to do that.

Reply:
wow shorter, faster, stronger